### PR TITLE
increase memory limits for kube-dns

### LINKED ISSUE #204: kube-dns pods crash consistently
kube-dns pods crash consistently because of too small memory limits (they are running in hyperd). we should increase the memory limits for it, e.g.

```sh
kubectl -n kube-system patch deployment kube-dns -p '{"spec":{"template":{"spec":{"containers":[{"name":"kubedns","resources":{"limits":{"memory":"256Mi"}}},{"name":"dnsmasq","resources":{"limits":{"memory":"128Mi"}}},{"name":"sidecar","resources":{"limits":{"memory":"64Mi"}}}]}}}}'
```

### PR DESCRIPTION
**Pull request by feiskyer**
Fix #204.

### CODE REVIEW

**Comment by feiskyer** [comment]
cc/ @resouer

File: cluster/allinone.sh [hunk 1]
```diff
@@ -154,6 +154,8 @@ setup-master() {
     kubectl taint nodes --all node-role.kubernetes.io/master-
     # approve kublelet's csr for the node.
     kubectl certificate approve $(kubectl get csr | awk '/^csr/{print $1}')
+    # increase memory limits for kube-dns
+    kubectl -n kube-system patch deployment kube-dns -p '{"spec":{"template":{"spec":{"containers":[{"name":"kubedns","resources":{"limits":{"memory":"256Mi"}}},{"name":"dnsmasq","resources":{"limits":{"memory":"128Mi"}}},{"name":"sidecar","resources":{"limits":{"memory":"64Mi"}}}]}}}}'
```
Comment (resouer) [inline cluster/allinone.sh]: why not change frakti's default resource configure?

Comment (feiskyer) [inline cluster/allinone.sh]: default resource configure is applied for all pods, while we only need this for kube-dns.

**Comment by resouer** [comment]
lgtm

**Pull request merged**